### PR TITLE
[ci] elpi 1.12

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

**Comment by gares** [comment]
as soon as docker ends I'll rebase.
@coq/ci-maintainers please review the last commit and assign

File: dev/ci/docker/bionic_coq/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# CACHEKEY: "bionic_coq-V2020-11-24-V91"
+# CACHEKEY: "bionic_coq-V2020-11-26-V92"
```
Comment (SkySkimmer) [inline dev/ci/docker/bionic_coq/Dockerfile:1]: The number is supposed to be random not incremental btw
but we really should change the scheme to be something like "$date-(short hash of dockerfile)" (which means stop copying it in the dockerfile ofc)

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
What milestone do you want for this?

File: dev/ci/docker/bionic_coq/Dockerfile (hunk 1 above)
Comment (gares) [reply dev/ci/docker/bionic_coq/Dockerfile:1]: ok

**Comment by gares** [comment]
I need to backport this, since the windows installer needs elpi 1.12

File: dev/ci/docker/bionic_coq/Dockerfile (hunk 1 above)
Comment (gares) [reply dev/ci/docker/bionic_coq/Dockerfile:1]: A file containing its own hash is not so easy to do ;-)

**Comment by SkySkimmer** [comment]
@coqbot: merge now

**Comment by gares** [comment]
super!

**Pull request merged**